### PR TITLE
Logic separation

### PR DESCRIPTION
**Pull request by nhurtado**
The main file was separated into multiple files.

### CODE REVIEW

**Comment by nhurtado** [comment]
[Separate program's logic](https://trello.com/c/oCoQWUHi/12-separate-programs-logic)

File: MineMap.py [hunk 1]
```diff
@@ -324,25 +257,17 @@ def total_mineral_weight_block_model():
                     elif grade_type == 2:
                         grade_weight += block_grades[block_grade]["value"] * block.weight
                     elif grade_type == 3:
-                        grade_weight += block_grades[block_grade]["value"] * block.weight / 32000
+                        grade_weight += block_grades[block_grade]["value"] * block.weight / 35273.962
                     elif grade_type == 4:
                         grade_weight += block_grades[block_grade]["value"] * block.weight * 0.0001
-        print("Total weight of {} is {} tons.".format(grade, grade_weight))
+        print("Total weight of {} is {} metric tons.".format(grade, grade_weight))
```
Comment (vreyn) [inline MineMap.py]: The calculation of this (and the other statistics) shouldnt be mixed with printing to console. This is part of the idea of domain isolation. The domain knows about statistics but shouldnt have to know directly about how the statistics will be shown

**Comment by vreyn** [comment]
I reviewed only this final PR which I think has all the other commits.

File: MineMap.py (hunk 1 above)
Comment (nhurtado) [reply MineMap.py]: Pushed the requested changes to the branch.

**Pull request merged**